src/dredd-command.coffee

```diff
@@ -281,7 +296,8 @@ class DreddCommand
       return if @finished
 
     configurationForDredd = @initConfig()
-    @dreddInstance = @initDredd configurationForDredd
+    @logDebuggingInfo(configurationForDredd)
```

Comment:
Is it intentional to output the debugging info always? It seems to me with every Dredd run, debug info will be printed.